src/html-conventions-lib/connectors-with-same-name.xsl

```diff
@@ -0,0 +1,118 @@
+            </xsl:if>
+        </xsl:for-each>
+    </xsl:template>
+
+
+
+
+
+    <xd:doc>
+        <xd:desc>Check the multiplicity values from a group of connectors with same name</xd:desc>
+        <xd:param name="connectorName"/>
+        <xd:param name="root"/>
+    </xd:doc>
+    <xsl:template name="multiplicityForConnectorsWithSameName">
+        <xsl:param name="connectorName"/>
+        <xsl:param name="root"/>
+        <xsl:variable name="connectorsWithSameName"
+            select="f:getConnectorByName($connectorName, $root)"/>
+        <xsl:variable name="multiplicityValues"
+            select="$connectorsWithSameName/*[role/@name = $connectorName]/type/@multiplicity"/>
+        <xsl:variable name="allConnectorsHaveMultiplicityValue"
+            select="fn:count($connectorsWithSameName) = fn:count($multiplicityValues)"/>
+        <xsl:sequence
+            select="
+                if (f:areStringsEqual($multiplicityValues) and $allConnectorsHaveMultiplicityValue) then
+                    ()
+                else
+                    f:generateHtmlWarning('The multiplicity value used is not equal for all the connectors with this name')"
+        />
+    </xsl:template>
+
+
+    <xd:doc>
+        <xd:desc>Check the definition values from a group of connectors with same name</xd:desc>
+        <xd:param name="connectorName"/>
+        <xd:param name="root"/>
+    </xd:doc>
+    <xsl:template name="definitionForConnectorsWithSameName">
+        <xsl:param name="connectorName"/>
+        <xsl:param name="root"/>
+        <xsl:variable name="connectorsWithSameName"
+            select="f:getConnectorByName($connectorName, $root)"/>
+        <xsl:variable name="definitionValues"
+            select="$connectorsWithSameName/*[role/@name = $connectorName]/../documentation/@value"/>
+
+        <xsl:variable name="descriptionsWithAnnotations" as="xs:string*"
+            select="
+                for $connector in $connectorsWithSameName
+                return
+                    if ($connector/documentation/@value) then
+                        fn:concat($connector/documentation/@value, ' (', f:getConnectorName($connector), ') ')
+                    else
+                        ()"/>
+
+        <xsl:variable name="allConnectorsHaveDefinition"
+            select="fn:count($connectorsWithSameName) = fn:count($definitionValues)"/>
+        <xsl:sequence
+            select="
+                if (f:areStringsEqual($definitionValues) and fn:boolean($definitionValues and $allConnectorsHaveDefinition)) then
+                    f:generateHtmlInfo(fn:concat('All the connectors with this name have the same definition. ',
+                    'Here is the usage: ',
+                    fn:string-join($descriptionsWithAnnotations, ',')))
+                else
+                    if (fn:boolean($definitionValues)) then
+                        f:generateHtmlWarning(fn:concat('The definition for the connectors with this name is different. ',
```

Comment:
The message is as follows: 

When a property is reused in multiple contexts, the meaning given by the definition is expected to be the same. In this case, multiple definitions are found.
_The text of the definition 1 in italic_ 
_(the connection name 1 in brackets and new line and italic)_
_The text of the definition 1 in italic and on a new line_ 
_(the connection name 2 in brackets and new line and italic)_
... etc.

Reply:
You can't have HTML markup in a string, sequence, select statement or fn:concat